client/components/ContributorsList.tsx

```diff
@@ -29,8 +30,8 @@ export function ContributorsList({
             className="relative"
           >
             <Image
-              src={contributor.avatar}
-              alt={contributor.name}
+              src={"https://dummyimage.com/600x400/000000/ffffff"}
+              alt={"contributor.name"}
```

Comment:
do you need quotes here?

Reply:
This is temporary hardcoded alt to avoid error, will add real after connecting gravatar